kin-sdk-core/build.gradle

```diff
@@ -31,12 +31,15 @@ android {
 }
 
 dependencies {
-    implementation 'com.github.kinfoundation:android-stellar-sdk:master-SNAPSHOT'
-    implementation supportDependencies.appCompat
+    implementation 'com.github.kinfoundation:android-stellar-sdk:transaction_increase_timeout_fix_error_handling-SNAPSHOT'
```

Comment:
why did you change it to: `transaction_increase_timeout_fix_error_handling`
I would also add it to `dependencies.gradle`

Reply:
I have PR for the stellar-sdk, let's review it together with this one, so it will get to master/dev together